qa/common/trace_summary.py

```diff
@@ -389,7 +389,10 @@ def find_first_id_with_tensor(traces):
 
 def get_numpy_array(tensor):
     dtype = TRITON_TYPE_TO_NUMPY[tensor["dtype"]]
-    value = map(float, tensor["data"].split(","))
+    if dtype == np.object_:
+        value = next(csv.reader([tensor["data"]], skipinitialspace=True))
```

Comment:
Would something like `value = map(str, tensor["data"].split(","))` work here instead? This would align more closely with the existing code.

Calling `next` just once to retrieve a single element from an iterator seems unintuitive here. Can `csv.reader(...)` yield an iterator for multiple elements for example?

Reply:
Or even just `value = tensor["data"].split(",")` ?

Reply:
Using split doesn’t handle the scenario where the comma delimiter is inside a quoted string which is why I am using the `CSV`